### ISSUE TITLE
New 'Copy emails from WorkFlow Module' option to Workflow's 'Create Record' action

### ISSUE THREAD

**Issue by adriangibanelbtactic** [salesagility/SuiteCRM#1294]
We would like Suitecrm AOW module to have a new option in 'Create Record' action so that we are able to copy emails from the "WorkFlow Module" to the bean which it's being created.

We have already implemented this functionality ourselves and we think it would be a great addition to SuiteCRM.

I have opened a pull request about it: https://github.com/salesagility/SuiteCRM/pull/1293 and I would like to obtain feedback about it so that I can fix / improve the code so that it gets accepted upstream.

Thank you very much!

**Comment by samus-aran** [comment]
@adriangibanelbtactic Thanks for your contribution.  Usually with enhancements we ask you make a PR to develop rather than master.  Can this be amended and we will consider all PR for the next major release?

**Comment by adriangibanelbtactic** [comment]
I have added another pull request: https://github.com/salesagility/SuiteCRM/pull/1348 done against develop branch as requested.

Can you reopen the bug so that you can give me feedback on this PR?

Sorry if I'm doing things twice but somehow I suspect I need to open both a PR and an issue.

**Comment by samus-aran** [comment]
@adriangibanelbtactic NP, I'll reopen the issue.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [New 'Copy emails from WorkFlow Module' option to Workflow's 'Create Record' action](https://trello.com/c/fndr2BJ1/102-new-copy-emails-from-workflow-module-option-to-workflow-s-create-record-action)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
